projects/addon-editor/extensions/iframe-editor/iframe-editor.component.ts

```diff
@@ -0,0 +1,59 @@
+import {DOCUMENT} from '@angular/common';
+import {ChangeDetectionStrategy, Component, Inject, Self} from '@angular/core';
+import {DomSanitizer, SafeResourceUrl} from '@angular/platform-browser';
+import {AbstractTuiEditorResizable} from '@taiga-ui/addon-editor/components/editor-resizable';
+import {TuiDestroyService} from '@taiga-ui/cdk';
+
+import {
+    TUI_IFRAME_EDITOR_OPTIONS,
+    TuiEditableIframe,
+    TuiEditableIframeOptions,
+} from './iframe-editor.options';
+
+@Component({
+    selector: 'tui-iframe-editor',
+    templateUrl: './iframe-editor.component.html',
+    styleUrls: ['./iframe-editor.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [TuiDestroyService],
+})
+export class TuiIframeEditorComponent extends AbstractTuiEditorResizable {
+    get attrs(): TuiEditableIframe {
+        return (this.node?.attrs as unknown as TuiEditableIframe) || {src: ''};
+    }
+
+    get src(): SafeResourceUrl {
+        return this.sanitizer.bypassSecurityTrustResourceUrl(this.attrs.src ?? '');
+    }
+
+    get width(): number | string | null {
+        return this._width || this.attrs.width || null;
+    }
+
+    get height(): number | string | null {
```

Comment:
maybe we should move getters `attrs`,  `width`, `height` inside `AbstractTuiEditorResizable` ? 🤔

Reply:
(any descendant class class simply override any of them, if there is need)

Reply:
fixed